weasyl/submission.py

```diff
@@ -1072,7 +1077,7 @@ def select_recently_popular():
             LEFT JOIN favorite ON favorite.type = 's' AND submission.submitid = favorite.targetid
         WHERE
             submission.unixtime > EXTRACT(EPOCH FROM now() - %(max_days)s * INTERVAL '1 day')::INTEGER AND
```

Comment:
Could this be
WHERE
    submission.submitid >= (SELECT cutoff FROM t)?

That would also simplify the favorite where clause.